samples/net/sockets/echo_client/README.rst

```diff
@@ -90,6 +90,36 @@ The certificate and private key used by the sample can be found in the sample's
 :ref:`sockets-echo-server-sample` enable establishing a secure connection
 between the samples.
 
+SOCKS5 proxy support
+====================
+
+It is also possible to connect to the echo-server through a SOCKS5 proxy.
+To enable it, use ``-DOVERLAY_CONFIG=overlay-socks5.conf`` when running ``west
+build`` or  ``cmake``.
+
+By default, to make the testing easier, the proxy is expected to run on the
+same host as the echo-server in Linux host.
+
+To start a proxy server, ``ssh`` can be used.
+Use the following command to run it on your host with the default port:
+
+.. code-block: console
+
+        $ ssh -N -D 0.0.0.0:1080 localhost
+
+.. code-block: console
+
+        $ ssh -N -D [::]:1080 localhost
```

Comment:
I'd suggest to explicitly say this command is for proxy on IPv6 address. And also, say explicitly of both commands need to be run at the same time (I for one would run only one for IPv4 for starters).